docs/snippets/ov_extensions.cpp

```diff
@@ -67,6 +69,27 @@ class CustomOp : public ov::op::Op {
     std::shared_ptr<ov::Node> clone_with_new_inputs(const ov::OutputVector&) const override { return nullptr; }
 };
 
+//! [frontend_extension_framework_map_CustomElu]
+class CustomElu : public ov::op::Op {
+private:
+    float m_alpha;
+    float m_beta;
+
+public:
+    OPENVINO_OP("CustomElu");
+
+    bool visit_attributes(ov::AttributeVisitor& visitor) override {
+        visitor.on_attribute("alpha", m_alpha);
+        visitor.on_attribute("beta", m_beta);
+        return true;
+    }
+
+    // ... implement other required methods
+//! [frontend_extension_framework_map_CustomElu]
+    std::shared_ptr<ov::Node> clone_with_new_inputs(const ov::OutputVector& inputs) const override { return nullptr; }
```

Comment:
I believe it will not show up in the document, but why not to put the correct code here anyway? Requires a ctor (two extra lines) and correct (one-line) implementation in this particular method. I mean that we shouldn't write code that is not correct and should be never used. And this return of `nullptr` is not correct.

Reply:
done